lib-compose/src/main/java/com/what3words/components/compose/maps/state/camera/W3WGoogleCameraState.kt

```diff
@@ -1,27 +1,27 @@
 package com.what3words.components.compose.maps.state.camera
 
+import androidx.annotation.UiThread
 import androidx.compose.runtime.Immutable
 import com.google.android.gms.maps.CameraUpdateFactory
 import com.google.android.gms.maps.model.CameraPosition
 import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.LatLngBounds
 import com.google.maps.android.compose.CameraPositionState
 import com.what3words.core.types.geometry.W3WCoordinates
 import com.what3words.core.types.geometry.W3WRectangle
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.launch
 
 @Immutable
 class W3WGoogleCameraState(override val cameraState: CameraPositionState) :
     W3WCameraState<CameraPositionState> {
 
     companion object {
-        const val MY_LOCATION_ZOOM = 19f
+        const val MY_LOCATION_ZOOM = 20f
     }
 
     override var gridBound: W3WRectangle? = null
 
-    override fun orientCamera() {
+    @UiThread
```

Comment:
The @UiThread annotation in Android is a contract or indicator, not a guarantee and it does not enforce it at runtime or switch to the main thread automatically. To ensure thread safety and compliance with coroutine rules, please modify the function to explicitly run on the Dispatchers.Main dispatcher